### ISSUE TITLE
Import Events from Alternatives to Meetup, Like Luma

### ISSUE THREAD

**Issue by allella** [hackgvl/hackgreenville-com#215]
TLDR;  Meetup.com, apparently owned by WeWork, changed their pricing model and it's likely to change the way free groups look at the value of that service.  The founder of FreeCodeCamp started a conversation about making a containerized, host-your-own meetup event platform to offer a free open-source alternative.

*** Note: Chapter is an archived project. Though, we may still try to provide support for GetTogether or other FOSS tools.

- https://twitter.com/ossia/status/1183845054449930241
- https://discordapp.com/channels/633401573002969128/633401573002969132
- https://github.com/freeCodeCamp/chapter

**Comment by allella** [comment]
Matt [mentioned someone forked GetTogether](https://github.com/GetTogetherComm/GetTogether/issues/323#issuecomment-1998505017) after not succeeding in taking over that repo and project

**Comment by allella** [comment]
[Robert mentioned Luma](https://github.com/hackgvl/hackgreenville-com/issues/240#issuecomment-2179322235), which has been getting traction, but it seemed like someone posted in the Slack that the pricing is almost the same outside of the free level.

**Comment by allella** [comment]
Matt Thornton mentioned https://www.radius.to/  - free tier, but no higher tier pricing listed and not a working app

**Comment by allella** [comment]
@bogdankharchenko here's an example of the JSON+LD payload on an organization page.  

This uses https://schema.org/Event and some of the fields are also Schema.org, like


[eventStatus](https://schema.org/eventStatus) values are of type [EventStatusType](https://schema.org/EventStatusType) with possible values

- EventCancelled
- EventMovedOnline
- EventPostponed
- EventRescheduled
- EventScheduled


[eventAttendanceMode](https://schema.org/OfflineEventAttendanceMode) is one of these [EventAttendanceModeEnumeration](https://schema.org/EventAttendanceModeEnumeration) values to indicate in-person, online, or both

- OfflineEventAttendanceMode
- OnlineEventAttendanceMode
- MixedEventAttendanceMode

A list of local orgs we've seen using Luma is in the previous comment and the below markup is for https://lu.ma/pixelpushers

```
{
  "@type": "Organization",
  "name": "Pixel Pushers",
  "url": "https://lu.ma/pixelpushers",
  "description": "Join us once a quarter for Greenville's web creator meetup.",
  "image": [
    "https://social-images.lu.ma/cdn-cgi/image/format=auto,fit=cover,dpr=2,background=white,quality=75,width=500,height=500/api/calendar-social-image?calendar_avatar_url=https%3A%2F%2Fimages.lumacdn.com%2Fcalendars%2Frz%2F0ca757e9-59c1-43e0-826c-a7a9ff2a5221&calendar_geo_city=Greenville&calendar_name=Pixel%20Pushers&calendar_tint_color"
  ],
  "location": {
    "@type": "Place",
    "address": {
      "@type": "PostalAddress",
      "addressLocality": "Greenville",
      "addressCountry": "United States",
      "addressRegion": "South Carolina"
    },
    "geo": {
      "@type": "GeoCoordinates",
      "latitude": "34.852618",
      "longitude": "-82.394010"
    },
    "latitude": "34.852618",
    "longitude": "-82.394010"
  },
  "potentialAction": {
    "@type": "SubscribeAction",
    "actionStatus": "PotentialActionStatus",
    "url": "https://lu.ma/pixelpushers"
  },
  "events": [
    {
      "@type": "Event",
      "@id": "https://lu.ma/4robg3w4",
      "name": "Pixel Pushers",
      "eventAttendanceMode": "https://schema.org/OfflineEventAttendanceMode",
      "eventStatus": "https://schema.org/EventScheduled",
      "location": {
        "@type": "Place",
        "name": "Register to See Address",
        "address": "Register to See Address"
      },
      "image": [
        "https://images.lumacdn.com/cdn-cgi/image/format=auto,fit=cover,dpr=2,background=white,quality=75,width=500,height=500/event-covers/xs/735b0447-080d-439b-a885-07e93c5523db"
      ],
      "description": "Pixel Pushers",
      "startDate": "2024-08-19T18:30:59.726-04:00",
      "endDate": "2024-08-19T20:30:59.726-04:00",
      "offers": [
        {
          "@type": "Offer",
          "url": "https://lu.ma/4robg3w4",
          "name": "General Admission",
          "price": 0,
          "priceCurrency": "usd",
          "availability": "https://schema.org/InStock"
        }
      ],
      "organizer": [
        {
          "@type": "Organization",
          "name": "Benten Woodring",
          "url": "https://lu.ma/user/usr-KZhxhrbzjGxrFrS"
        }
      ],
      "performer": [
        {
          "@type": "Organization",
          "name": "Benten Woodring",
          "url": "https://lu.ma/user/usr-KZhxhrbzjGxrFrS"
        }
      ]
    }
  ],
  "@context": "https://schema.org"
}
```





















[JSON Pretty Print](http://jsonprettyprint.net/) - [Privacy Policy](http://jsonprettyprint.net/privacy)`

**Comment by allella** [comment]
Per #341 here are the calendar IDs for the API path Bogdan figured out the calendar API URL and the calendar IDs are visible. This is not the same as the paid API, but rather is a public API used by the web app that's callable from a GET without an API key.

The calendar IDs start with "cal-" and can be found by clicking the iCal icon and viewing the URL provided by "Copy URL to Clipboard", or viewing the source code and find the calendar_api_id, or for organizations without a custom URL slug, the cal-id is in the main org page's URL, as with Zapier, below.

## Luma Future Events by Organization

**Pixelpushers**
https://lu.ma/pixelpushers
Calendar ID: cal-kmfIhQsKoWIuNab
https://api.lu.ma/calendar/get-items?calendar_api_id=cal-kmfIhQsKoWIuNab&period=future&pagination_limit=50


**Tech Taco Tuesday PHP Meetup**
https://lu.ma/nullai
Calendar ID: cal-p2JSHSmjI4ustkj
https://api.lu.ma/calendar/get-items?calendar_api_id=cal-p2JSHSmjI4ustkj&period=future&pagination_limit=50

**Zapier Customer Meetup**
https://lu.ma/calendar/cal-4XXrm7XCZBXRMbc
Calendar ID: cal-4XXrm7XCZBXRMbc
https://api.lu.ma/calendar/get-items?calendar_api_id=cal-4XXrm7XCZBXRMbc&period=future&pagination_limit=50

## Luma Past Events
Set _period=past_ to view past events
https://api.lu.ma/calendar/get-items?calendar_api_id=cal-4XXrm7XCZBXRMbc&period=past&pagination_limit=50

**Comment by allella** [comment]
Pixel Pushers is now live

https://hackgreenville.com/orgs/pixel-pushers

Joel said the Zapier meetup may not be a recurring thing.

I'll check with Kevin Dees about the name of the group he's running.
